cmd/nerdctl/create_test.go

```diff
@@ -0,0 +1,33 @@
+/*
+   Copyright The containerd Authors.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+package main
+
+import (
+	"testing"
+
+	"github.com/containerd/nerdctl/pkg/testutil"
+)
+
+func TestCreate(t *testing.T) {
+	t.Parallel()
+	base := testutil.NewBase(t)
+	tID := testutil.Identifier(t)
+
+	base.Cmd("create", "--name", tID, testutil.CommonImage, "echo", "foo").AssertOK()
+	defer base.Cmd("rm", "-f", tID).Run()
+	base.Cmd("ps", "-a").AssertOutContains("Created")
```

Comment:
Please also test that the container can be started with "start"

Reply:
@AkihiroSuda Yes, I am still waiting for https://github.com/containerd/nerdctl/pull/647 to merge